prac_06/guitars.py

```diff
@@ -0,0 +1,32 @@
+from prac_06.guitar import Guitar
+
+
+def main():
+    guitars = []
+    print("My guitars!")
+    guitar_name = input("Guitar name: ")
+    while guitar_name != "":
+        guitar_year = int(input("Guitar year: "))
+        guitar_cost = float(input("Guitar cost: $ "))
+        new_guitar = [guitar_name, guitar_year, guitar_cost]
+        guitars.append(new_guitar)
+        print(f"{new_guitar[0]} ({new_guitar[1]}) : ${new_guitar[2]} added.")
+        guitar_name = input("Guitar name: ")
```

Comment:
We know these are all in context of a guitar. "Guitar_name" could be "name" and so on?